test/utils/matchers.js

```diff
@@ -1,43 +1,43 @@
-beforeEach(function() {
+beforeEach(() => {
   jasmine.addMatchers({
-    toHaveNoDataAttributes: function() {
+    toHaveNoDataAttributes() {
       return {
-        compare: function(actual) {
-          var dataAttrs = [];
-          for (var i = 0; i < actual.attributes.length; i++) {
+        compare(actual) {
+          let dataAttrs = []
+          for (let i = 0; i < actual.attributes.length; i++) {
             if ((/data\-/).test(actual.attributes[i].nodeName)) {
-              dataAttrs.push(actual.attributes[i].nodeName);
+              dataAttrs.push(actual.attributes[i].nodeName)
             }
           }
-          var pass = dataAttrs.length === 0;
+          const passed = dataAttrs.length === 0
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>Missing semicolon.